components/Disclaimer/index.tsx

```diff
@@ -0,0 +1,20 @@
+import Link from 'next/link'
+import { StyledDisclaimer } from './styled'
+
+function Disclaimer() {
+  return (
+    <StyledDisclaimer>
+      By connecting a wallet, you agree to Nine Chronicles Ltd's{' '}
+      <Link href="/docs/terms" target="_blank" rel="noopener">
```

Comment:
```suggestion
      By connecting a wallet, you agree to Planetarium Labs{"'"}
```

ESLint rule react/no-unescaped-entities is causing build failure, so I modified it.

however I think we should discuss this disclamer text with @lounlee?

Reply:
sorry, Nine Chronicles Ltd was correct. then escaping `Ltd's` to `Ltd&apos;s` will enough.